tests/sles4sap/crash/crash_azure.pm

```diff
@@ -0,0 +1,139 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary:  Executes a crash scenario on azure.
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use mmapi 'get_current_job_id';
+use serial_terminal 'select_serial_terminal';
+use version_utils 'is_sle';
+use sles4sap::azure_cli;
+use utils 'script_retry';
+
+use constant DEPLOY_PREFIX => 'clne';
+
+sub run {
+    my ($self) = @_;
+
+    die('Azure is the only CSP supported for this test')
+      unless check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE');
+
+    my $rg = DEPLOY_PREFIX . get_current_job_id();
+
+    select_serial_terminal;
+
+    my $provider = $self->provider_factory();
+
+    my $os_ver;
+    if (get_var('PUBLIC_CLOUD_IMAGE_LOCATION')) {
+        # This section is only needed by Azure tests using images uploaded
+        # with publiccloud_upload_img.
+        $os_ver = $self->{provider}->get_blob_uri(get_var('PUBLIC_CLOUD_IMAGE_LOCATION'));
+    } else {
+        $os_ver = get_required_var('CLUSTER_OS_VER');
+    }
+
+    # remove configuration file created by the PC factory
+    # as it interfere with ssh behavior.
+    assert_script_run('rm ~/.ssh/config');
```

Comment:
Are you sure about it? We already have a mechanism to select a specific ssh config file for each test using `  PUBLIC_CLOUD_SSH_CONFIG: 'publiccloud/ssh_config_sap'`